### ISSUE TITLE
Using `\` to make newlines doesn't work

### ISSUE THREAD

**Issue by Moosems** [littlewhitecloud/TkTerminal#19]

**Comment by littlewhitecloud** [comment]
?

**Comment by Moosems** [comment]
In a terminal, you can use backslashes to give multiple lines for one big command.

**Comment by littlewhitecloud** [comment]
@Moosems can you give a screenshot? Or can you assign yourself to improve this function? I really can't image how to use it.

**Comment by littlewhitecloud** [comment]
Is it just like:
```batch
cl /Ox /Ot /favor:blend /EHsc \
/GA /nologo /LD /DEF: plugin.def plugin.c \
/w /errorReport:none /std:c17 /D "_WINDOWS"
```

**Comment by Moosems** [comment]
Yeah. In windows you use `&&` but on many other systems and with bash you use `\`. I'll make an implementation sometime. I'm busy this week though so I can't do nearly and programming right now.

**Comment by littlewhitecloud** [comment]
fixed it in #31

**Comment by Moosems** [comment]
Also needs && for windows ;).

**Comment by littlewhitecloud** [comment]
> Also needs && for windows ;).

Added it! :D

**Comment by Moosems** [comment]
You're epic.

**Comment by littlewhitecloud** [comment]
> You're epic.

Thanks! Actally it is quite easy to add "&&"!
This is my first time to work with others on my first repo. I feel very accomplished. It is good for me to work with a funny guy like you. You are epic too!

**Comment by littlewhitecloud** [comment]
So, what's wrong with the latest code?

**Comment by littlewhitecloud** [comment]
![image](https://github.com/littlewhitecloud/TkTerminal/assets/71159641/2a0f2f6a-e49d-486f-840c-d73fb32d3ea6)
I tested a lot but found nothing

**Comment by Moosems** [comment]
The backslash isn't removed so the terminal treats the chars as escaped chars.

**Comment by littlewhitecloud** [comment]
![image](https://github.com/littlewhitecloud/TkTerminal/assets/71159641/0cd7f9ca-dba4-4dac-ba43-706282fe1195)
What?

**Comment by littlewhitecloud** [comment]
Can you give a screenshot? @Moosems

**Comment by littlewhitecloud** [comment]
Can you give a screenshot or an example? @Moosems

**Comment by Moosems** [comment]
I'm not with my computer this week but from memory here's what happens:
```console
Moosems$ cd \
.
Unable to change to directory (insert escaped period char)
```

**Comment by littlewhitecloud** [comment]
Let me test it.

**Comment by littlewhitecloud** [comment]
![image](https://github.com/littlewhitecloud/TkTerminal/assets/71159641/47a7181c-fd8b-41a8-9eef-bc436895d5e8)

**Comment by littlewhitecloud** [comment]
quite strange

**Comment by littlewhitecloud** [comment]
> I'm not with my computer this week but from memory here's what happens:
> 
> ```
> Moosems$ cd \
> .
> Unable to change to directory (insert escaped period char)
> ```

It should not be, I write
```python
self.longcmd += cmd.split(self.longsymbol)[0]
```
to cut off the `\\`

**Comment by littlewhitecloud** [comment]
Still keep open it.
And try to fix it when @Moosems you have computer and you are free to test.

**Comment by Moosems** [comment]
Will do!
